LocalSettings.php

```diff
@@ -4335,6 +4335,12 @@
 	'wmgWikibasePropertyNamespaceID' => [
 		'default' => 120
 	],
+	'wmgWikibaseRepoItemNamespaceID' => [
```

Comment:
Please It should be "0" now

Reply:
The default is being maintained. It will be changeable from ManageWiki once this is merged and live.

Reply:
Okay then
On manage wiki It's actually 0 but it's that $wgWBRepoSettings['entityNamespaces']['item'] = NS_MAIN; that needs to be configured